perses/analysis/load_simulations.py

```diff
@@ -199,101 +238,91 @@ def sample_history(self,method='binding'):
     def reset_history(self):
         self.count = 0
 
-
-def kcal_to_kt(x):
-    """
-    This should be deleted and just use the simtk units protocol
-    :param x: float, energy in kcal
-    :return: float, energy in kT
-    """
-    # TODO remove this
-    return x*1.688
-
-def get_experimental(molecules, i,j):
-    """ Determine experimental relative free energy from two experimntal absolute results
-
-    Parameters
-    ----------
-    molecules : list
-        list of load_simulation.molecule objects
-    i : int
-        index of first molecule
-    j : int
-        index of second molecule
-
-    Returns
-    -------
-    tuple
-        relative free energy and associated error
-
-    """
-    moli = molecules[i]
-    molj = molecules[j]
-    ddG = moli.exp - molj.exp
-    ddG_err = moli.experr - molj.experr
-    return (ddG, ddG_err)
-
-
-def load_experimental(exp_file):
-    """ Load details from a freesolv database.txt-like file
-
-    Parameters
-    ----------
-    exp_file : str
-        path to text file
-
-    Returns
-    -------
-    list
-        list of load_simulation.molecule objects, contained in the textfile
-
-    """
-    molecules = []
-    with open(exp_file) as f:
-        for i, line in enumerate(f):
-            molecules.append(molecule(i, line))
-    return molecules
-
-def run(molecules,simtype='sams',offline_freq=10):
-    """Load the simulation data for a set of molecules, for both forward and backward simulations
-
-    Parameters
-    ----------
-    molecules : list
-        list of load_simulation.molecule objects, of which to find simulation data for
-    simtype : type
-        Description of parameter `simtype`.
-    offline_freq : type
-        Description of parameter `offline_freq`.
-
-    Returns
-    -------
-    type
-        Description of returned object.
-
-    """
-    import itertools
-    import os
-
-    n_ligands = len(molecules)
-    all_simulations = []
-    for a, b in itertools.combinations(range(0, n_ligands), 2):
-        path = f'lig{a}to{b}'
-        if os.path.isdir(path) == True:
-            sim = simulation(a, b)
-            all_simulations.append(sim)
-        else:
-            print(f'Output directory lig{a}to{b} doesnt exist')
-
-        # now run the opposite direction
-        path = f'lig{b}to{a}'
-        if os.path.isdir(path) == True:
-            sim = simulation(b, a)
-            all_simulations.append(sim)
-        else:
-            print(f'Output directory lig{b}to{a} doesnt exist')
-
-    return all_simulations
-
-if __name__ == '__main__':
-    run(sys.argv[1])
+#def get_experimental(molecules, i,j):
```

Comment:
Feel free to delete this code if we don't need it. We can always come back and rescue it if we need it.